From b8b1dc36bdba9ff069b04919664131a87cbb19e7 Mon Sep 17 00:00:00 2001
From: Ilya Shlyakhter <ilya_shl@alum.mit.edu>
Date: Fri, 31 May 2019 15:01:38 -0400
Subject: [PATCH] added mach_voucher_types include

---
 coregrind/m_syswrap/syswrap-darwin.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/coregrind/m_syswrap/syswrap-darwin.c b/coregrind/m_syswrap/syswrap-darwin.c
index 590493d..cdc242e 100644
--- a/coregrind/m_syswrap/syswrap-darwin.c
+++ b/coregrind/m_syswrap/syswrap-darwin.c
@@ -67,6 +67,7 @@
 /* --- !!! --- EXTERNAL HEADERS start --- !!! --- */
 #include <mach/mach.h>
 #include <mach/mach_vm.h>
+#include <mach/mach_voucher_types.h>
 #include <semaphore.h>
 /* --- !!! --- EXTERNAL HEADERS end --- !!! --- */
 
-- 
2.20.1

