From 65e8b0aef3d1f364fbc795c9382aead166ea0724 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Julian=20R=C3=BCth?= <julian.rueth@fsfe.org>
Date: Wed, 18 Aug 2021 22:48:01 +0200
Subject: [PATCH] search for kernel defs in our sysroot

---
 coregrind/Makefile.am | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/coregrind/Makefile.am b/coregrind/Makefile.am
index 65d2c70..3d7f375 100644
--- a/coregrind/Makefile.am
+++ b/coregrind/Makefile.am
@@ -157,10 +157,10 @@ mach_hdrs += \
         m_mach/thread_act.h \
         m_mach/vm_map.h
 mach_defs += \
-	@XCODE_DIR@/mach/mach_vm.defs \
-        @XCODE_DIR@/mach/task.defs \
-        @XCODE_DIR@/mach/thread_act.defs \
-        @XCODE_DIR@/mach/vm_map.defs
+	${CONDA_BUILD_SYSROOT}/usr/include/mach/mach_vm.defs \
+        ${CONDA_BUILD_SYSROOT}/usr/include/mach/task.defs \
+        ${CONDA_BUILD_SYSROOT}/usr/include/mach/thread_act.defs \
+        ${CONDA_BUILD_SYSROOT}/usr/include/mach/vm_map.defs
 endif
 
 # Be careful w.r.t. parallel builds.  See section 27.9 of the automake info
-- 
2.33.0

